soc/nxp/imxrt/imxrt118x/soc.c

```diff
@@ -269,6 +269,14 @@ static ALWAYS_INLINE void clock_init(void)
 
 #endif /* CONFIG_COUNTER_MCUX_GPT */
 
+#if !(DT_NODE_HAS_COMPAT(DT_CHOSEN(zephyr_flash), nxp_imx_flexspi_nor)) &&              \
+	defined(CONFIG_MEMC_MCUX_FLEXSPI) && DT_NODE_HAS_STATUS(DT_NODELABEL(flexspi), okay)
+	/* Configure FLEXSPI1 using MuxOscRc24M */
+	rootCfg.mux = kCLOCK_FLEXSPI1_ClockRoot_MuxOscRc24M;
```

Comment:
Just a question here, why are we using the 24MHz oscillator for the FLEXSPI? Don't we want a higher speed clock for better performance? On other platforms the FLEXSPI has typically been clocked at 100 MHz or higher.

Reply:
This configuration does not affect the flexspi speed for XIP, if it is XIP, flexspi will be configured to 133MHz in boot_config on the RT11XX and RT118X platforms. Here configuration for flexspi only be used when application level need to use flexspi peripheral.(Align with the flexspi configuration in SDK)

Reply:
Ok, I was not aware we were aligning with the SDK clock configuration. I guess my question then becomes are we aware why the *SDK* is choosing such a low frequency when not running in XIP mode? This is a non-blocking question to be clear. I just want to make sure we have a reason here, since this code is at the SOC level and will affect all customer boards based on this SOC.

Reply:
Sorry, with confirmed with SDK release owner today, there is no issue on clock configuration on RT1180, default using Pll3Pfd0 in SDK clock_config.c file. I have updated and aligned with SDK on zephyr code. When porting RT1180 flexspi IP, I refer RT1170 clock code and use a low speed mux as clock source for RT1180. So the issue you raised is occurred on RT1170, however, the root course why using a so low speed mux as clock source, I don't know yet.